File: URI_1153.cs
using System; 

class URI {

    static void Main(string[] args) { 

       int x = int.Parse(Console.ReadLine());
        int multi=1;
        for (int i = 1; i <=x ; i++) {

            multi = multi*i;
        }
        Console.WriteLine(multi);

    }

}

File: URI_1153.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
    int x;
    cin>>x;
        int multi=1;
        for (int i = 1; i <=x ; i++) {

            multi = multi*i;
        }
        cout<<multi<<endl;
 
    return 0;
}
